platform/ios/app/MBXViewController.m

```diff
@@ -40,6 +40,7 @@ + (void)initialize
             @"MBXUserTrackingMode": @(MGLUserTrackingModeNone),
             @"MBXShowsUserLocation": @NO,
             @"MBXDebug": @NO,
+            @"MGLMapboxMetricsTestServerURL": @"https://cloudfront-staging.tilestream.net",
```

Comment:
Since you're already renaming the default, how about replacing "MapboxMetrics" with "Telemetry"?